### PR TITLE
Login adjustment for Web Version

### PR DESCRIPTION
**Pull request by mustiwebp**

### CODE REVIEW

File: lib/screens/login_screen.dart [hunk 1]
```diff
@@ -177,21 +186,29 @@ class _LoginScreenState extends State<LoginScreen>
                 width: 200,
                 child: Row(
                   children: <Widget>[
-                    Text(AppLocalizations.of(context)!.rememberMe),
+                    !kIsWeb
+                        ? Text(AppLocalizations.of(context)!.rememberMe)
+                        : const SizedBox(
+                            width: 0,
+                          ),
```
[the remark this answers is not in the document]
Comment (mustiwebp) [reply lib/screens/login_screen.dart:193]: I think it is not redundant because we need an else-case.

[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/login_screen.dart:189]: yes, we need an else case if the else case does anything.
This line does nothing.

[the remark this answers is not in the document]
Comment (mustiwebp) [reply lib/screens/login_screen.dart:189]: The colon is expected in this kind of if condition.

[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/login_screen.dart:189]: right, then you don't have to use colon.

**Pull request merged**